src/essreflectometry/calibrations.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
 import scipp as sc
 
-from .supermirror.types import Alpha, CriticalEdge, MValue, SupermirrorCalibrationFactor
+from .supermirror import Alpha, CriticalEdge, MValue, SupermirrorCalibrationFactor
```

Comment:
```suggestion
from . import supermirror
```
since `calibrations.Alpha` is a bit meaningless?